js/dnd.js

```diff
@@ -0,0 +1,81 @@
+jQuery(document).ready(function($) {
+
+    var dnd = {
+        handleSortable: function() {
+
+            var fixHelper = function(e, ui) {
+              ui.children().each(function() {
+                $(this).width($(this).width());
+              });
+              return ui;
+            };
+
+            $("#cop-sortable-table #the-list").sortable({
```

Comment:
well, that selector could be non-sense but it just indicates #the-list is inside #cop-sortable-table, for me, hierarchy matters! but ok.